hack/build-cross.sh

```diff
@@ -33,6 +33,12 @@ os::build::build_binaries "${OS_CROSS_COMPILE_TARGETS[@]}"
 OS_BUILD_PLATFORMS=("${OS_IMAGE_COMPILE_PLATFORMS[@]-}")
 os::build::build_static_binaries "${OS_IMAGE_COMPILE_TARGETS[@]-}" "${OS_SCRATCH_IMAGE_COMPILE_TARGETS[@]-}"
 
+# Build dockerregistry
+(
```

Comment:
@smarterclayton You mean changes in `hack/build-cross.sh ` or directory in general?